README.md

```diff
@@ -1 +1,170 @@
-# spello
\ No newline at end of file
+<p align="center">
+    <img src="logo.png" width="350", height="220">
```

Comment:
Change text in logo to "A Fast and accurate SpellCorrector"

Reply:
Done